sdk/tools/azure-openrewrite/src/test/resources/migrationExamples/azure-core/sample16/v1/RequestOptionsUse.java

```diff
@@ -6,19 +6,19 @@ public static void main(String... args) {
 
         // Sample 1: Basic usage
         RequestOptions options1 = new RequestOptions();
-        options1.addHeader("Custom-Header", "CustomValue");
-        options1.addHeader("Another-Header", "AnotherValue");
+        options1.setHeader("Custom-Header", "CustomValue");
+        options1.setHeader("Another-Header", "AnotherValue");
 
-        options1.addHeader(HttpHeaderName.CONTENT_TYPE, "application/json");
-        options1.addHeader(HttpHeaderName.ACCEPT, "application/json");
+        options1.setHeader(HttpHeaderName.CONTENT_TYPE, "application/json");
+        options1.setHeader(HttpHeaderName.ACCEPT, "application/json");
 
         options1.addQueryParam("queryParam1", "value1")
             .addQueryParam("queryParam2", "value2")
             .addQueryParam("queryParam3", "value3");
 
         RequestOptions options2 = new RequestOptions()
-            .addHeader(HttpHeaderName.CONTENT_TYPE, "application/json")
-            .addHeader(HttpHeaderName.ACCEPT, "application/json")
+            .setHeader(HttpHeaderName.CONTENT_TYPE, "application/json")
```

Comment:
We should still support `addHeader` functionality that is available in v1. In v2, the transformation might require a bit more changes to use the callback.